docs/runtime_CLI.md

```diff
@@ -317,6 +320,99 @@ the meter type is packets, the rate is entered in packets/microsecond and
 burst_size is the number of packets.
 
 
+### mirroring_add, mirroring_add_mc, mirroring_delete, mirroring_get
+
+When any of these events happened during packet processing:
+
++ Your P4_16 v1model architecture program called `clone` or `clone3`
+  extern functions during ingress or egress processing, and now
+  ingress or egress processing has completed
++ Your P4_14 program called `clone_ingress_pkt_to_egress` during
+  ingress processing, which is now complete
++ Your P4_14 program called `clone_egress_pkt_to_egress` during egress
+  processing, which is now complete
+
+then not only will the current "original" packet continue with
+whatever it normally does, as if your program had not called a clone
+operation, but the PRE (Packet Replication Engine) part of
+`simple_switch` will also create zero or more copies, or clones, of
+the current packet, and cause them to be enqueued in the packet buffer
+such that later each of the cloned packet(s) will perform egress
+processing.  Each of the clones is processed independently from each
+other, and independently from the original packet.
+
+Search for occurrences of the word "clone" in [this
+article](https://github.com/p4lang/behavioral-model/blob/master/docs/simple_switch.md#pseudocode-for-what-happens-at-the-end-of-ingress-and-egress-processing)
+for more details on how `simple_switch` deals with cloned packets.
+
+TBD: How many clone sessions are supported by `simple_switch`, without
+using any special default compilation options nor modifications to the
+source code?
```

Comment:
@antoninbas Do you happen to know the answer to this?  I can search through the code to find out, if. you prefer.  Also, I believe it is correct that clone session id 0 is not supported in the simple_switch data plane, but good if you can double-check me on that.

Reply:
You're the one who wrote that code Andy: https://github.com/p4lang/behavioral-model/blob/master/targets/simple_switch/register_access.h#L46 :)
* 0 is a valid  clone session id for bmv2 (so it can be used with the CLI) but not for P4Runtime. This is because we use an extra bit to encode whether the packet is being cloned.
* because the validity bit + the session id use 16 bits, that leaves us with 15 bits for the session id so valid values are in the range `[0, 32768]`.

Reply:
I have updated that text to give the range of clone session ids supported as 0 up to 32,767.  Thanks for reminding me :-)